### ISSUE TITLE
assignment with user-identity

### ISSUE THREAD

**Issue by robfrut135** [Azure/azure-cli-extensions#2381]
## Describe the bug

**Command Name**
`az blueprint assignment update
Extension Name: blueprint. Version: 0.1.1.`

**Errors:**
```
Unable to build a model: Unable to deserialize response data. Data: ['/subscriptions/9f89d8cc-c662-4695-b270-57c92157bdb5/resourceGroups/hip-networking/providers/Microsoft.ManagedIdentity/userAssignedIdentities/azure-blueprints-cross-subscription'], {UserAssignedIdentity}, TypeError: string indices must be integers, DeserializationError: Unable to deserialize response data. Data: ['/subscriptions/9f89d8cc-c662-4695-b270-57c92157bdb5/resourceGroups/hip-networking/providers/Microsoft.ManagedIdentity/userAssignedIdentities/azure-blueprints-cross-subscription'], {UserAssignedIdentity}, TypeError: string indices must be integers
Traceback (most recent call last):
...
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az blueprint assignment update --name {} --location {} --blueprint-version {} --locks-mode {} --parameters {} --description {} --subscription {} --identity-type {} --user-assigned-identities {}`

## Environment Summary
```
Linux-5.4.0-47-generic-x86_64-with-debian-bullseye-sid
Python 3.6.5
Installer: DEB

azure-cli 2.3.1 *
azure-cli-ml 1.3.0

Extensions:
blueprint 0.1.1
synapse 0.3.0
azure-cli-ml 1.3.0

```

**Comment by yonzhan** [comment]
blueprint

**Comment by yungezz** [comment]
hi @fengzhou-msft could you pls help to have a look?

**Comment by fengzhou-msft** [comment]
The issue is fixed in blueprint 0.2.0, please update and retry.

**Comment by robfrut135** [comment]
Great!!! it works :)
